bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/handler/PowermaxBridgeHandler.java

```diff
@@ -560,6 +576,14 @@ private synchronized void updateChannelsFromAlarmState(String channel, PowermaxS
         }
     }
 
+    /**
+     * Update all channels to a NULL state to indicate that communication with the panel is offline
+     */
+    private synchronized void setAllChannelsOffline() {
+        getThing().getChannels().forEach(c -> updateState(c.getUID(), UnDefType.NULL));
+        getThing().getThings().forEach(t -> t.getChannels().forEach(c -> updateState(c.getUID(), UnDefType.NULL)));
```

Comment:
Please do that in class PowermaxThingHandler method initializeThingState(ThingHandler bridgeHandler, ThingStatus bridgeStatus)